Prefabs/CameraRig/TrackedAlias/SharedResources/Scripts/TrackedAliasFacade.cs

```diff
@@ -94,5 +87,26 @@ public class TrackedAliasFacade : MonoBehaviour
         /// Retreives all of the linked CameraRig Right Controller Velocity Trackers.
         /// </summary>
         public List<VelocityTracker> RightControllerVelocityTrackers => cameraRigs.Select(rig => rig.RightControllerVelocity).Where(value => value != null).ToList();
+
+        /// <summary>
+        /// Emitted when the headset starts tracking for the first time.
+        /// </summary>
+        public UnityEvent HeadsetTrackingBegun = new UnityEvent();
+        /// <summary>
+        /// Emitted when the left controller starts tracking for the first time.
+        /// </summary>
+        public UnityEvent LeftControllerTrackingBegun = new UnityEvent();
+        /// <summary>
+        /// Emitted when the right controller starts tracking for the first time.
+        /// </summary>
+        public UnityEvent RightControllerTrackingBegun = new UnityEvent();
+
+        #region Internal Settings

```

Comment:
According to the conventions properties come **after** fields.

Reply:
this is different because it's the internal settings so it's best to just have it completely out of the way at the bottom of the inspector window